### PR TITLE
Create jdhzzz.json new user location :)

### PR DESCRIPTION
**Pull request by jdhzzz**
Here I am.

No changes to existing files.  Adding this location to the mix.

Note: @githubteacher

Closes #3026 

-Jeff

### CODE REVIEW

**Comment by jdhzzz** [comment]
this looks great 👍 you are my hero 🍕 .

**Comment by qjubic** [comment]
@jdhzzz 
silently cheering at random people 🎉  - don't mind the disturbance. 😄

File: _pins/jdhzzz.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+githubHandle:jdhzzz
```
Comment (crichID) [inline _pins/jdhzzz.json]: @jdhzzz the reason the tests are failing is that you must have a space after the `:` on each line. If you click the Files changed tab, you will see a small pencil at the top right of the file. Click that and you will be back in edit mode! Please let me know if I can help!

File: _pins/jdhzzz.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+latitude:42.639765
+longitude:-72.884955
+---
+<block end>
```
Comment (crichID) [inline _pins/jdhzzz.json]: @jdhzzz please remove this line!

**Comment by crichID** [comment]
@jdhzzz I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/3026/commits/5f69e23635395e5c8a9f99e8fd92944b8d37291d. Please take a quick look at the changes. Once the tests have finished running, please feel free to click the big green **Merge pull request** button. If you haven't merged it by February 3rd, I will go ahead and do it just to keep the repository clean!

**Pull request merged**